### PR TITLE
[Term Entry] Php Math Function: log10()

### PR DESCRIPTION
**Pull request by wejdan01**
### Description

-Added the log10 Term for Php
-Closes issue #2271 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/log10/log10.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+The **`log10()`** function returns the base-10 logarithm of a number.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/log10/log10.md:15]:
```suggestion

The **`log10()`** function returns the base-10 logarithm of a number.
```
Add line break.

File: content/php/concepts/math-functions/terms/log10/log10.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Syntax
+
+```pseudo
+log10($arg);
+```
+## Parameters:
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/log10/log10.md:22]:
```suggestion

```

File: content/php/concepts/math-functions/terms/log10/log10.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+---
+The **`log10()`** function returns the base-10 logarithm of a number.
+
+## Syntax
+
+```pseudo
+log10($arg);
+```
+## Parameters: 
+The function accepts a single parameter $arg which refers to the number of whose the logarithm is to be calculated.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/log10/log10.md:23]:
```suggestion
The function accepts a single parameter `$arg` which refers to the number of whose the logarithm is to be calculated.
```

File: content/php/concepts/math-functions/terms/log10/log10.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Codebyte Example
+
+Using `log10()` to return the abase-10 logarithm of `100`:
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/log10/log10.md:27]:
```suggestion
Using `log10()` to return the base-10 logarithm of `100`:
```

File: content/php/concepts/math-functions/terms/log10/log10.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+```codebyte/php
+<?php
+   $arg=100;
+   echo "log10(" . $arg. ")=" . log10($arg) . "
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/log10/log10.md:32]:
```suggestion
   echo "log10(" . $arg . ")=" . log10($arg) . "
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@wejdan01 I did a first review and left a few comments.

**Comment by wejdan01** [comment]
@SSwiniarski done

File: content/php/concepts/math-functions/terms/log10/log10.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+The **`log10()`** function returns the base-10 logarithm of a number.
+
+## Syntax
+
+```pseudo
+log10($arg);
+```
+
+## Parameters
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/log10/log10.md:24]: I wanted you to remove this Parameters header so the following sentence is part of the Syntax section.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@wejdan01 One more edit I wanted you to make.

File: content/php/concepts/math-functions/terms/log10/log10.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+```pseudo
+log10($arg);
+```
+
+The function accepts a single parameter `$arg` which refers to the number of whose the logarithm is to be calculated.
+
+## Codebyte Example
+
+Using `log10()` to return the base-10 logarithm of `100`:
```
Comment (KTom101) [inline content/php/concepts/math-functions/terms/log10/log10.md:28]:
```suggestion
The following codebyte example uses the `log10()` function to return the base-10 logarithm of `100`, then it outputs the result using `[echo](https://www.codecademy.com/resources/docs/php/string-functions/echo)`:
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@wejdan01 I added a couple of changes to this.

**Review by KTom101 — APPROVED** [review]
@wejdan01 Since a good amount of time has passed, I've gone ahead and added the changes myself in commit [ef80478](https://github.com/Codecademy/docs/pull/2287/commits/ef80478b36c8b9035d6ca567cdf207ab21841a79) & [24e72fd](https://github.com/Codecademy/docs/pull/2287/commits/24e72fd8e25800fd0c45086df77ebda33a3a15b3).

**Comment by KTom101** [comment]
@wejdan01 Entry is here: https://www.codecademy.com/resources/docs/php/math-functions/log10

You're currently listed as anonymous:

Screenshot 2023-06-26 at 7 12 07 PM

If you want to remain anonymous, there is nothing more you need to do. If you wish to be listed as a contributor, please follow the instructions linked [here](https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036).

**Pull request merged**